kratos/python/add_properties_to_python.cpp

```diff
@@ -147,10 +149,37 @@ bool HasTableHelperFunction1( TContainerType& rContainer,
     return rContainer.HasTable(XVar, YVar);
 }
 
+template<typename TVariableType>
+void AddInterfaceToAccessorFold(pybind11::class_<Properties, Properties::Pointer, Properties::BaseType>& pyProperties) {
+
+    pyProperties
+    .def("GetAccessor", [](Properties &rProperties, Variable<TVariableType> &rVariable) { 
+            auto accessor = &rProperties.pGetAccessor(rVariable);
+            
+            KRATOS_ERROR_IF(*accessor == nullptr) << "Trying to get a consumed or invalid Accessor." << std::endl;
+                
+            return accessor; 
+        }, py::return_value_policy::reference_internal)
+    .def("SetAccessor", [](Properties &rProperties, Variable<TVariableType> &rVariable, std::unique_ptr<Accessor>& rpAccessor) {
+            if (rpAccessor == nullptr)
```

Comment:
cosmetic comment, but there is an officil way of doing this:

https://en.cppreference.com/w/cpp/memory/unique_ptr/operator_bool